packages/web/src/app/shell/page-with-sidenav/page-with-sidenav.component.scss

```diff
@@ -18,26 +18,21 @@
 
 .page-container {
   display: grid;
-  margin: 0 104px;
-  max-width: var(--max-desktop-width);
-  padding-right: min(80px, 4vw); // This shifts the content to the right to make it feel more centered
+  padding: 0 calc(min(80px, 4vw) + 104px) 0 104px; // This shifts the content to the right to make it feel more centered
```

Comment:
Tiny thing, but this can be defined as: `padding-inline: 104px calc(min(80px, 4vw) + 104px);`

Reply:
However, its tomato tomato, so you be the judge 😄 Approved!

Reply:
Nice point!🤩